itou/approvals/models.py

```diff
@@ -575,9 +575,9 @@ class Suspension(models.Model):
     """
 
     # Min duration: none.
-    # Max duration: 12 months (could be adjusted according to user feedback).
-    # 12-months suspensions can be consecutive and there can be any number of them.
-    MAX_DURATION_MONTHS = 12
+    # Max duration: 36 months (could be adjusted according to user feedback).
+    # 36-months suspensions can be consecutive and there can be any number of them.
+    MAX_DURATION_MONTHS = 36
```

Comment:
Le commit : Maximum suspension is now of **24** months.
Le code : Max duration: **36** months
Le relecteur : 
![d97fca4c05f75f6bb2952eb621556732_w200](https://user-images.githubusercontent.com/20045330/216367877-b6656ac0-a2a4-4c68-9ad9-81216c051833.gif)

Reply:
ahahaha

Reply:
Ben oui 12+24=36 DUH.

Reply:
La durée était de 12 mois, maintenant c'est 36.

Reply:
👀 
![image](https://user-images.githubusercontent.com/10533583/216581921-999314be-58b1-483f-ba10-afa29ee873b2.png)